src/main/host/syscall/protected.h

```diff
@@ -69,7 +69,7 @@ struct _SysCallHandler {
  * send/recv or read/write syscall. It would be more efficient to dynamically
  * compute how much we can read/write rather than using this static size.
  * TODO: remove this when we switch to dynamic size calculations. */
-#define SYSCALL_IO_BUFSIZE (1024 * 16) // 16 KiB
+#define SYSCALL_IO_BUFSIZE (1024 * 1024 * 10) // 10 MiB
```

Comment:
Did this end up mattering for correctness? Worth a comment if so

Reply:
I'm pretty sure it didn't matter, but I left it in since we wanted it anyway.